### ISSUE TITLE
Browse mode: When tabbing through wordpress.com, when an iFrame is encountered, NVDA speaks the content when using IE whereas it does not in Firefox

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3611]
**Reported by nvdakor on 2013-10-28 10:57**
Hi,
A Korean user reports the following:
When you tab through wordpress.com, when NVDA encounters an iFrame with Twitter and Facebook like buttons, NVDA:
- Speaks the content of the frame when using Internet Explorer.
- Does not speak the content of the frame when browsing with Firefox.

STR:
1. From both IE and Firefox, go to http://wordpress.org/
2. Press TAB to navigate through the page until you hear "privacy policy" and "GPL2" links.
3. Press TAB once  to access the social networking iFrame.
expected: NVDA speaks the content of the frame such as Twitter button and Facebook's like button (works when browsing with IE).
Actual: In Firefox, NvDA does not announce the content of the frame.

Specs:
- Internet Explorer: 10.0.
- Firefox: 24.0 (this problem happens with 27.0 nightly, according to the original reporter).
  Thanks.

**Comment by bhavyashah** [comment]
Confirmed. Using Firefox 55 stable and NVDA 2017.3, here's an interesting log snippet present while I Tab through the social media links (which a user might not even realize is a frame since NVDA appears not to report it):
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.249):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.265):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.279):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.296):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.296):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.312):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.326):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.326):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.342):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IAccessibleRole (21:28:26.342):
accRole failed: (-2147024809, 'The parameter is incorrect.', (None, None, None, 0, None))
With reference to @jcsteh's https://github.com/nvaccess/nvda/issues/3611#issuecomment-155311495, has anyone got any clues as to what on earth might be causing this? :)

**Comment by Adriani90** [comment]
Actually I cannot reproduce this in Firefox 63.0.3.
@advck1123, @bhavyashah can you still reproduce it with NVDA 2018.4 RC1?

**Comment by Adriani90** [comment]
@bhavyashah are you still available?

**Comment by Adriani90** [comment]
I am closing this as works for me. I cannot reproduce it in Firefox 66.0.5.
